portfolio/src/main/webapp/scripts/helpers.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @param {number} radians
+ * @param {number} radius
+ */
+function getCircleX(radians, radius) {
+  return Math.cos(radians) * radius;
+}
+
+/**
+ * @param {number} radians
+ * @param {number} radius
+ */
+function getCircleY(radians, radius) {
+  return Math.sin(radians) * radius;
+}
+
+/** @param {number} number */
+const convertNumberToPixelString = (number) => {
+  return `${number}px`;
+}
+
+/**
+ * @param {string} contentType
+ * @param {boolean} isVisible
+ */
+const toggleContent = (contentType, isVisible) => {
```

Comment:
misleading variable name as this is the ID of a single element ("content type" implies it's referring to all elements of a particular type).